src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -8543,15 +8547,16 @@ void MacroAssembler::char_array_compress(Register src, Register dst, Register le
     jcc(Assembler::zero, post_alignment);
```

Comment:
@RogerRiggs I think you changed the wrong line 😉 
```suggestion
    jccb(Assembler::zero, post_alignment);
```

Reply:
Thanks for spotting that mistake, corrected in [b2fc385](https://github.com/openjdk/jdk/pull/16425/commits/b2fc38550ba95bcd7ec1ae4f52f22b220afcb045).